src/Item_SoftwareLicense.php

```diff
@@ -113,6 +113,52 @@ public function rawSearchOptions()
         return $tab;
     }
 
+    /**
+     * display a value according to a field
+     *
+     * @since 11.0
+     *
+     * @param $field     String         name of the field
+     * @param $values    String / Array with the value to display
+     * @param $options   Array          of option
+     *
+     * @return string
+     **/
+    public static function getSpecificValueToDisplay($field, $values, array $options = [])
+    {
+        if (!is_array($values)) {
+            $values = [$field => $values];
+        }
+
+        switch ($options['searchopt']['id']) {
+            case '164':
+                $softlicense = new SoftwareLicense();
+                $softlicense->getFromDB($options['raw_data']['id']);
+                $assign_item = self::countForLicense($options['raw_data']['id']);
+                $remaining_item = $softlicense->fields['number'] - $assign_item;
+                $total = __('Total') . ': ' . $softlicense->fields["number"];
+                $assigned = __('Assigned') . ': ' . $assign_item;
+                $remaining = __('Remaining') . ': ' . $remaining_item;
+                $span_class = $remaining_item >= 0 ? '#74b72e' : '#cf9b9b';
```

Comment:
Use variable names that correspond to their contents.
Here it's a color, not a span class.
```suggestion
                $color = $remaining_item >= 0 ? '#74b72e' : '#cf9b9b';
```